### PR TITLE
Issue #15007: Add input file for instanceof operator in OperatorWrap

### LINKED ISSUE #15007: Add Check Support for Java 21 Record Pattern Syntax: OperatorWrap
child of https://github.com/checkstyle/checkstyle/issues/14890
I have read check documentation: https://checkstyle.org/checks/whitespace/operatorwrap.html#OperatorWrap
I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

```
PS D:\CS\test> cat src/Test2.java                                                
record ColoredPoint(int p, int x, String c) { }
record Rectangle(ColoredPoint upperLeft, ColoredPoint lowerRight) { }
public class Test2 {
    void test(Object obj) {
        if (!(obj instanceof                        // violation
                Rectangle(ColoredPoint record,ColoredPoint yield))) { 
        }
    }

}
PS D:\CS\test> cat config.xml                                                    
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
        "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
    <property name="charset" value="UTF-8"/>
    <module name="TreeWalker">
        <module name="OperatorWrap">

        </module>

    </module>
</module>
PS D:\CS\test> java  -jar checkstyle-10.17.0-all.jar -c config.xml src/Test2.java
Starting audit...
[ERROR] D:\CS\test\src\Test2.java:5:19: 'instanceof' should be on a new line. [OperatorWrap]
Audit done.
Checkstyle ends with 1 errors.
PS D:\CS\test> 

```
Describe what you expect in detail.

This can be closed without any updates

### PR DESCRIPTION
**Pull request by mahfouz72**
closes #15007

### CODE REVIEW

**Comment by mahfouz72** [comment]
github, generate site

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/operatorwrap/InputOperatorWrapInstanceOfOperator.java
Comment (nrmancuso) [inline src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/operatorwrap/InputOperatorWrapInstanceOfOperator.java:1]: Please add a variation on each method here where we correctly line wrap the `instanceof`. Example:
```
        if (o
              instanceof Integer i) {}


```

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/operatorwrap/InputOperatorWrapInstanceOfOperator.java
Comment (mahfouz72) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/operatorwrap/InputOperatorWrapInstanceOfOperator.java:1]: done

**Review by nrmancuso — APPROVED** [review]

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/OperatorWrapCheck.java [hunk 1]
```diff
@@ -31,7 +31,13 @@
 
 /**
  * <p>
- * Checks the policy on how to wrap lines on operators.
+ * Checks the policy on how to wrap lines on
+ * <a href ="https://docs.oracle.com/javase/tutorial/java/nutsandbolts/opsummary.html">
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/OperatorWrapCheck.java:35]: Remove space before `=`.

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/operatorwrap/InputOperatorWrapInstanceOfOperator.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
+/*
+OperatorWrap
+option = (default)nl
```
Comment (rnveach) [inline src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/operatorwrap/InputOperatorWrapInstanceOfOperator.java:3]: Duplicate file with other option.

**Review by rnveach — CHANGES_REQUESTED** [review]

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/OperatorWrapCheck.java (hunk 1 above)
Comment (mahfouz72) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/OperatorWrapCheck.java]: done

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/operatorwrap/InputOperatorWrapInstanceOfOperator.java (hunk 2 above)
Comment (mahfouz72) [reply src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/whitespace/operatorwrap/InputOperatorWrapInstanceOfOperator.java:3]: done

**Review by rnveach — APPROVED** [review]

**Pull request merged**